ginga/rv/plugins/Drawing.py

```diff
@@ -455,6 +474,44 @@ def scale_object(self, w):
         delta = float(w.get_text())
         self.canvas.edit_scale(delta, delta, self.fitsimage)
 
+    def import_regions_cb(self, w):
+        if not ap_region.HAVE_REGIONS:
+            self.fv.show_error("Please install astropy regions to use this",
+                               raisetab=True)
+            return
+        from ginga.gw.GwHelp import FileSelection
+        fs = FileSelection(w.get_widget(), all_at_once=True)
+        fs.popup('Load regions file', self._import_regions_files,
+                 initialdir='.',
+                 filename='Region files (*.reg)')
+
+    def _import_regions_files(self, paths):
+        for path in paths:
+            objs = ap_region.import_ds9_regions(path)
+            for obj in objs:
+                self.canvas.add(obj, redraw=False)
+
+        self.canvas.update_canvas()
+
+    def export_regions_cb(self, w):
+        if not ap_region.HAVE_REGIONS:
+            self.fv.show_error("Please install astropy regions to use this",
+                               raisetab=True)
+            return
+        fs = Widgets.SaveDialog('Save Regions', selectedfilter='*.reg')
+        path = fs.get_path()
+        if path is None:
+            # cancelled
+            return
+
+        regs = ap_region.export_regions_canvas(self.canvas)
+
+        # TODO: allow other export formats that regions allows
+        format = 'ds9'
+
+        # dialogs confirms if they want to overwrite
```

Comment:
Check for the dialog when you actually try the PR.  Pretty sure I encountered one because it is built in to the save file dialog that you implemented a long time ago.

Reply:
Oh yeah, I see it. 😸